cli/flox-core/src/activations.rs

```diff
@@ -124,7 +125,7 @@ pub struct Activation {
     ///
     /// The activation should not be cleaned up until all PIDs have exited or
     /// expired.
-    attached_pids: Vec<AttachedPid>,
+    pub attached_pids: Vec<AttachedPid>,
```

Comment:
I'd prefer we keep this private to the struct to go retain its "readonly" / controlled modification property.
If the field is public you'll have to reason about using the `Activation`s methid vs accessing the field explicitly.